Directory.Build.props

```diff
@@ -1,13 +1,13 @@
 <Project>
```

Comment:
Will this file impact our `tooling\Microsoft.VisualStudio.RazorExtension\Microsoft.VisualStudio.RazorExtension.csproj`?